### PR TITLE
Feature/AddChild

### PR DESCRIPTION
**Pull request by SandraCoburn**
# Description

Added functionality to AddChild component to connect to the backend. Added a new function in the api index to take care of the axios call to backend. 
Added FamilySetting component and connected to AddChild and ParentDashboard.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [X] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [X] No
- [ ] Not necessary

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/Help/Help.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          </div>
+        </Layout>
+      </Layout>
+    </>
+  );
+};
+
+export default ParentDashboard;
+
+// import React from 'react';
```
Comment (bramirez96) [inline src/components/pages/Help/Help.js:59]: Gotta drop these here comments!

**Review by bramirez96 — CHANGES_REQUESTED** [review]
Just gotta remove the comments!

File: src/components/pages/AddChild/AddChild.less [hunk 2]
```diff
@@ -50,7 +50,9 @@ input {
   margin: auto;
   // margin-top: 1%;
```
Comment (Schrese) [inline src/components/pages/AddChild/AddChild.less:51]: There's a commented out style left here

File: src/components/pages/Help/Help.js (hunk 1 above)
Comment (Schrese) [inline src/components/pages/Help/Help.js:59]: If this code isn't needed, then we might need to delete it

**Review by Nick-Ohman — APPROVED** [review]
looks great!

**Review by bramirez96 — APPROVED** [review]
Looks great!

**Review by Schrese — APPROVED** [review]

**Pull request merged**